ports/cppzmq/vcpkg.json

```
@@ -17,7 +17,13 @@
   ],
   "features": {
     "draft": {
-      "description": "Build and install draft"
+      "description": "Build and install draft",
+      "dependencies": [
+        {
+          "name": "zeromq",
+          "features": ["draft"]
+        }
+      ]
```

```suggestion
      "description": "Build and install draft",
      "dependencies": [
        {
          "name": "zeromq",
          "default-features": false,
          "features": [
              "draft"
          ]
        }
      ]
```
If only depend on feature `zeromq[draft]` then other features should be disabled to reduce build time.